alpha/lib/data/kBaseEntryCloneOptionComponent.php

```diff
@@ -0,0 +1,65 @@
+<?php
+/**
+ * @package Core
+ * @subpackage model.data
+ */
+class kBaseEntryCloneOptionComponent extends kBaseEntryCloneOptionItem
+{
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * @var int itemType - critera is a collection of properties of the entry. The list of properties of the same
+     *                      criteria are not always copied during the clone operation
+     */
+    protected $itemType;
```

Comment:
Please organize the code such that all the properties will be defined before the get\set